addons/l10n_lv/data/res_bank_data_lt.xml

```diff
@@ -0,0 +1,122 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo noupdate="1">
+
+    <record id="res_bank_CBSBLT26XXX" model="res.bank">

```

Comment:
Thanks so much for this. I see this is an update of the banks provided in `l10n_lt`.
Could we split this off into a separate PR that just modifies `l10n_lt/data/res_bank_data.xml`?